cards/multilang-financial-professional-location/component.js

```diff
@@ -6,9 +6,8 @@ class multilang_financial_professional_locationCardComponent extends BaseCard['m
   }
 
   onMount() {
-    const onInteractiveMapPage = !!document.querySelector('.js-answersInteractiveMap');
-    onInteractiveMapPage && new CardListenerAssigner({card: this}).addListenersToCard();
-    super.onMount();
+    const onVerticalFullPageMap = !!document.querySelector('.js-answersVerticalFullPageMap');
+    onVerticalFullPageMap && new CardListenerAssigner({card: this}).addListenersToCard();    super.onMount();
```

Comment:
Is it intentional that `super.onMount()` is on this line? It below this line for the other cards